src/ir/ty.rs

```diff
@@ -673,7 +673,7 @@ impl Type {
             CXType_ConstantArray => {
                 let inner = Item::from_ty(&ty.elem_type(), location, parent_id, ctx)
                                 .expect("Not able to resolve array element?");
-                TypeKind::Array(inner, ty.num_elements())
+                TypeKind::Array(inner, ty.num_elements().expect("Not an array?"))
```

Comment:
Please use `unwrap()`, if this eventually fails, it's easy to trace it through gdb, and we expect this to be infallible.

Thanks!